proteus/LinearAlgebraTools.py

```diff
@@ -50,11 +53,19 @@ def scatter_reverse_add(self):
 class ParVec_petsc4py(p4pyPETSc.Vec):
     """
     Parallel vector using petsc4py's wrappers for PETSc
-    """
-    def __init__(self,array=None,bs=None,n=None,N=None,nghosts=None,subdomain2global=None,blockVecType="simple"):
-        if array == None:
-            return#cek hack, don't know why init gets called by PETSc.Vec duplicate function
+    WIP -- This function builds the local to global mapping for the PETSc parallel vectors.  At this
+    point it only works when the variables can be interwoven (eg. stablized elements where velocity and
+    pressure come from the same space).  We would like to extend this functionality to include finite
+    element spaces that cannot be interwoven such as Taylor Hood.
```

Comment:
I need to remove this comment, since it's not true anymore. We can to interleave and non-interleaved now.